src/AttributesAndMethods.jl

```diff
@@ -221,13 +348,24 @@ struct picard_group
 end
 export picard_group
 
+
+"""
+    picard_group( v::NormalToricVariety )
+
+Computes the Picard group of the normal toric variety `v`.
+"""
 function picard_group( v::NormalToricVariety )
     gap_PicardGroup = GAP.Globals.PicardGroup( v.GapNTV )
     return picard_group( gap_PicardGroup )
 end
 export picard_group
 
 
+"""
+    name_of_variety( v::NormalToricVariety )
+
+Returns the name of the normal toric variety `v`, if set. Otherwise returns "No name set for this variety".
+"""
 function name_of_variety( v::NormalToricVariety )
     if ! ( Bool( GAP.Globals.HasNameOfVariety( v.GapNTV ) ) )
```

Comment:
There is no need for `Bool` here, you can just write `if ! GAP.Globals.HasNameOfVariety(v.GapNTV)`

Reply:
Thank you for the pointer. I will change this.